### DISCUSSION TITLE
Variables y declaraciones en Rust

### DISCUSSION THREAD

**Discussion by giscus** [RustLangES/blog#30] [Comentarios de Artículos]
# Variables y declaraciones en Rust

La declaración de variables es algo fundamental en el aprendizaje de Rust, en este articulo abordaremos una guía rápida. - By @Phosphorus-M

https://rustlanges.github.io/blog/articles/variables-y-declaraciones.html

**Comment by gixi404** [comment]
good

**Comment by Arg128** [comment]
Yess :)
